src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -1049,11 +1049,13 @@ registerVisualizationLayerDefinition({
       positionedGlyph.varGlyph.sources[sourceIndex].customData[
         "fontra.development.status"
       ];
-    if (status === undefined) {
-      return;
-    }
 
-    const color = [...statusFieldDefinitions[status].color];
+    const defaultStatusValue = statusFieldDefinitions.find(
+      (statusDef) => statusDef.isDefault
+    )?.value;
+    const statusValue = status !== undefined ? status : defaultStatusValue;
```

Comment:
Suggest to make `status` a `let`, and only look up the default value if it indeed is undefined. This would be faster in the case where status _isn't_ undefined.

You should probably also return if the default status value can't be found.

Reply:
Thanks. Done.